cookbook/how_to_apply_patch.rst

```diff
@@ -0,0 +1,123 @@
+.. index::
+    single: Patch
+
+How to apply patch
+==================
+
+The patch file name supports the following naming convention: {package}-{version_to_apply}.patch.
+For example, platform-1.9.2.patch.
+
+**1** Create backups of your Database and Code.
+
+**2** "Cd" to the crm root folder and switch the application to the maintenance mode.
+
+.. code-block:: bash
+
+    $ cd /path/to/application
+    $ sudo -uwww-data app/console lexik:maintenance:lock --env prod
+
+
+**3** Stop the cron tasks
```

Comment:
Should be checked is it necessary to stop the cron in case when we put system to the maintenance mode with app/console lexik:maintenance:lock. If I remember correctly cron will not perform any actions while maintenance mode is turned on.

According to this code https://github.com/orocrm/platform/blob/master/src/Oro/Bundle/CronBundle/Command/CronCommand.php#L45-L51 cron shouldn't run tasks in maintenance.